static/sass/landing.scss

```diff
@@ -77,7 +77,7 @@ section{
 	}
 
 }
-$next-section-color: $white;
+$next-section-color: $gray-lighter;
```

Comment:
Can't we make section colors alternate using a CSS selector like `element:nth-child(2n+1)`